mender-deb-package

```diff
@@ -167,6 +167,35 @@ prepare_recipe() {
     cp support/mender-monitor.service debian/
   fi
 
+  # 'armhf' can mean both ARMv7 (usually and in Debian, in particular) and ARMv6
```

Comment:
wait, in debian armhf always means one thing: armv7.  do we really support raspberry pi that runs armv6?

Reply:
No, but we used to. So if people try to use our `armhf` packages on 32-bit RPis, we should produce an error.